celery_worker.py

```diff
@@ -0,0 +1,29 @@
+from celery import Celery
+from config.logger_config import logger
+from config.redis_config import REDIS_HOST,REDIS_PORT,REDIS_DB
+
+# Convert Redis Port and DB to Integer
+REDIS_PORT = int(REDIS_PORT)
```

Comment:
Create a new class Config in a file config.py and put all configurations, secrets  in that file.